science/gmtsar/Portfile

```diff
@@ -0,0 +1,50 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+name                gmtsar
+version             5.6
+categories          science
+platforms           darwin
+license             GPL-3
+maintainers         ucsd.edu:dsandwell \
+                    ucsd.edu:xix016 \
+                    ucsd.edu:jdesanto \
+                    openmaintainer
+description         GMTSAR
```

Comment:
This `description` is too short; it doesn't really describe anything. What you've written in the `long_description` field is probably the right length for the `description` field; then I'd expand the `long_description` to a sentence or two.